src/nio/crypto/olm_machine.py

```
@@ -694,10 +695,45 @@ def _handle_key_claiming(self, response):
                         "Olm session."
                     )
 
+    def _handle_cross_signing_keys(
+        self,
+        key_dict: Dict,
+        key_class: Union[MasterKey, SelfSigningKey, UserSigningKey],
+        key_store: Dict,
+    ) -> None:
+        for user_id, key_payload in key_dict.items():
+            key_info = next(iter(key_payload.values()))
+            public_key = key_info["keys"][next(iter(key_info["keys"]))]
```

Ruff (0.12.2)

Local variable `public_key` is assigned to but never used

Remove assignment to unused variable `public_key`

(F841)